library/registry

```diff
@@ -1,8 +1,12 @@
 Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
-GitRepo: https://github.com/docker/distribution-library-image.git
+GitRepo: https://github.com/distribution/distribution-library-image.git
 GitFetch: refs/heads/master
 
+Tags: 3.0.0-beta.1
+Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
+GitCommit: a943e89c3efe06134cd9a4b439203c5341082cbc
+
 Tags: 3.0.0-alpha.1
```

Comment:
The release of `beta.1` means that `alpha.1` is now effectively unsupported, right? (ie, there won't ever be an `alpha.2` now that `beta.1` is out, right?)

So this section should be removed/replaced. :+1:

Reply:
Ok I binned that: https://github.com/docker-library/official-images/pull/17151/commits/698473001c2ae3277d810726e0be480261ad3667

Reply:
Good catch; thank you! 🙏